src/routes/goosepage/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Goosepage from './Goosepage';
+
+const title = 'The Goose Page';
+
+export default {
+
+    path: '/goosepage',
+
+        action() {
```

Comment:
You can run `npm run lint` before making every commit, it will allow you to avoid problems with code style. Also `cmd + alt + L` can save your time.